gcloud-java-core/src/main/java/com/google/gcloud/PageImpl.java

```diff
@@ -71,6 +71,7 @@ public PageImpl(NextPageFetcher<T> pageFetcher, String cursor, Iterable<T> resul
     this.results = results;
   }
 
+  @SuppressWarnings("unchecked")
```

Comment:
I think this may not be necessary if we change `Collections.EMPTY_LIST` to `Collections.emptyList()`